torch/csrc/jit/passes/liveness.cpp

```diff
@@ -30,6 +34,35 @@ struct LivenessAnalyzer {
     return result;
   }
 
+  // temporary make loop counts live for the duration of the loop
+  // as they are needed by BailOuts in the loop
+  void insertExplicitUsesOfLoopCounters(
+      Block* b,
+      std::vector<Node*>& counters) {
+    for (auto it : b->nodes()) {
+      if (it->kind() == prim::Loop) {
+        LoopView lv(it);
+        WithInsertPoint guard(*lv.bodyBlock()->nodes().end());
```

Comment:
I think this should be `lv.bodyBlock()`. Dereferencing `end()` shouldn't be considered defined.